src/ontology/mondo-ingest.Makefile

```diff
@@ -159,6 +159,17 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## No slurp outputs
Still trying to figure this out. When I run `migrate.py` manually, I get slurp outputs for `icd11foundation` no problem. However when I run `slurp-icd11foundation`, it gets `rm`'ed at the end, and I can't figure out what's causing that. But also I checked the `build-mondo-ingest` logs, and ICD11 isn't even getting run during that anyway, and I'm not sure why yet.

Reply:
Addressed by: https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1533102897